### ISSUE TITLE
Example Tetris doesn't compile

### ISSUE THREAD

**Issue by AurelienFT** [vlang/v#370]
I'm using Fedora and I can't compile the teris example the following error appear :
```
pass=1 fn=``
panic: gl_lin.v:5
redefinition of `gl.init_glad`
exit():  
```

**Comment by musou1500** [comment]
@AurelienFT it is probably duplicated. I posted a patch to issue below.
https://github.com/vlang/v/issues/355

**Comment by cdunn2001** [comment]
Same as #361 and #369. We need to chill out and wait for the fix in #355.

**Comment by AurelienFT** [comment]
Yeah okey wasn't the same error message so I posted it sorry.

I close it
